gamemaker.py

```diff
@@ -1,12 +1,24 @@
 class GameMaker:
   def __init__(self, database):
     self.database = database
-    self.loggedin = false
   
   def login(self, name, password):
-    if (self.database.gameMakerCred = {name, password}):
-      self.loggedin = true
-  //TODO Implement checkStatus
+    returnValue = ""
+    if (self.database.curUser != name && self.database.gameMakerCred = {name, password}):
+      self.database.curUser = name
+      returnValue = "User " + name + " logged in!"
```

Comment:
Are we returning this or just printing it? And if you are returning it, you forgot to return it.